### DISCUSSION TITLE
Show NodeToolbar on Custom Edge click

### DISCUSSION THREAD

**Discussion by ritesh-nahar** [xyflow/xyflow#3875] [Q&A]
I have create a custom edge and on hover I am showing "+" icon on it. Is there any possibility to show NodeToolbar on click of "+" icon and it should show the toolbar next to the edge?

**Comment by russiann** [comment]
I have the same scenario.
I need to show a toolbar on an Edge. 

Is there any way to use the `<NodeToolbar/>` as a "`<EdgeToolbar/>`" ?
If not, what's the better way to do that?
